lib/rspec_api_documentation/views/slate_example.rb

```diff
@@ -6,24 +6,18 @@ def initialize(example, configuration)
         self.template_name = "rspec_api_documentation/slate_example"
       end
 
-      def curl_with_linebreaks
-        requests.map {|request| request[:curl].lines }.flatten.map do |line|
-          line.rstrip.gsub("\t", '  ').gsub(' ', '&nbsp;').gsub('\\', '&#92;')
-        end.join "<br>"
-      end
-
-      def explanation_with_linebreaks
-        explanation.gsub "\n", "<br>\n"
-      end
-
       def write
```

Comment:
With the update to the SlateWriter, I don't think we need this method any more. I deleted it locally and everything passes.